src/main/java/fileprocessor/FindFiles.java

```diff
@@ -0,0 +1,55 @@
+package fileprocessor;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.java.Log;
+import lombok.val;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Log
+@RequiredArgsConstructor
+public class FindFiles implements Callable<List<File>> {
+
+    private final String directory;
+
+    @Override
+    public List<File> call() throws Exception {
+        File folder = new File(directory);
+
+        log.fine(STR."Folder: \{folder.getAbsolutePath()}");
+
+        val files = recursivelyGetFilesToScrape(folder).collect(Collectors.toList());
+        files.forEach(f -> log.finest(STR."Found file: \{f.getAbsolutePath()}"));
+        log.info(STR."Number of found files: \{files.size()}");
+        return files;
+    }
+
+    private Stream<File> recursivelyGetFilesToScrape(File folder) {
```

Comment:
I *assume* this is supposed to return *only* files, and not the folders those files are on. However at least on Windows (and accessing an SMB share), it returns the folders, and then the files in them.

```
2024-07-21 09:23:25 FINE fileprocessor.FindFiles call Folder: Z:\xxxx\anidbclitest1
2024-07-21 09:23:25 FINEST fileprocessor.FindFiles lambda$call$0 Found file: Z:\xxxx\anidbclitest1\A Silent Voice [anidb-10937]
2024-07-21 09:23:25 FINEST fileprocessor.FindFiles lambda$call$0 Found file: Z:\xxxx\anidbclitest1\Adesugata Mahou no Sannin Musume [anidb-3633]
2024-07-21 09:23:25 FINEST fileprocessor.FindFiles lambda$call$0 Found file: Z:\xxxx\anidbclitest1\Akikan! (2009) [anidb-7132]
2024-07-21 09:23:25 FINEST fileprocessor.FindFiles lambda$call$0 Found file: Z:\xxxx\anidbclitest1\Akuma-kun Youkoso Akuma Land e!! [anidb-2051]
2024-07-21 09:23:25 FINEST fileprocessor.FindFiles lambda$call$0 Found file: Z:\xxxx\anidbclitest1\A Silent Voice [anidb-10937]\A Silent Voice [BlurayDesuYo][BluRay][1920x1080][HEVC][7273A66D].mkv
2024-07-21 09:23:25 FINEST fileprocessor.FindFiles lambda$call$0 Found file: Z:\xxxx\anidbclitest1\A Silent Voice [anidb-10937]\A Silent Voice [Okay-Subs][BluRay][1920x1036][HEVC][6457E1C3].mkv
2024-07-21 09:23:25 FINEST fileprocessor.FindFiles lambda$call$0 Found file: Z:\xxxx\anidbclitest1\Adesugata Mahou no Sannin Musume [anidb-3633]\Adesugata Mahou no Sannin Musume [raw][unknown][320x240][MS MP4x][65E7ABA9].avi
2024-07-21 09:23:25 FINEST fileprocessor.FindFiles lambda$call$0 Found file: Z:\xxxx\anidbclitest1\Akikan! (2009) [anidb-7132]\Akikan! (2009) [TSSMA][DVD][848x480][h264][D4784111].mkv
2024-07-21 09:23:25 FINEST fileprocessor.FindFiles lambda$call$0 Found file: Z:\xxxx\anidbclitest1\Akuma-kun Youkoso Akuma Land e!! [anidb-2051]\Akuma-kun Youkoso Akuma Land e!! [Rododendron][www][1280x720][h264][648B02E2].mkv
2024-07-21 09:23:25 INFO fileprocessor.FindFiles call Number of found files: 9
```
Which then results in an fatal exception when it tries to hash the directory
```
2024-07-21 09:23:25 FINE processing.Mod_EpProcessing addFiles File Count changed to 9
2024-07-21 09:23:25 INFO processing.Mod_EpProcessing start Starting processing
2024-07-21 09:23:25 FINE processing.Mod_EpProcessing processEps Processing file Z:\xxxx\anidbclitest1\A Silent Voice [anidb-10937] with Id 0
java.io.FileNotFoundException: Z:\xxx\anidbclitest1\A Silent Voice [anidb-10937] (Access is denied)
        at java.base/java.io.FileInputStream.open0(Native Method)
        at java.base/java.io.FileInputStream.open(FileInputStream.java:213)
        at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)
        at processing.FileParser.run(FileParser.java:25)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1583)
```

Reply:
i think you had an older version, but tell me if that's wrong.
I forgot to check for directories in `shouldScrapeFile` (line 43). 
It currently should not return any directories, but if that doesn't work with SMB shares, let me know and i'll fix it